messaging/messaging-core/src/main/java/uk/gov/justice/services/messaging/Metadata.java

```diff
@@ -76,4 +77,5 @@
      */
     JsonObject asJsonObject();
 
+    Optional<ZonedDateTime> created();
```

Comment:
Do we want this to be consistently named - I think we called the builder method createdAt. Worth a quick team discussion before it is set in stone.